src/ontology/mappings/d3fend-cco.ttl

```diff
@@ -87,6 +87,61 @@ d3f:Technique rdfs:subClassOf cco:DirectiveInformationContentEntity.
 
 d3f:InformationContentEntity owl:equivalentClass cco:InformationContentEntity.
 
+d3f:Log rdfs:subClassOf cco:RepresentationalInformationContentEntity .
+
+# An example cco gives of a representational ICE is a transcript, which seems similar to a log.
+
+d3f:Sensor owl:equivalentClass cco:Sensor .
```

Comment:
I'll add this to the comment. The definition of sensor they give doesn't have this issue, but I can see your worry with cyber sensors. I don't think they're thinking of them as ICEs, since they're viewing them as performing operations, as opposed to being the information about those operations.